recipes/tensorly/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "tensorly" %}
+{% set version = "0.4.2" %}
+{% set fn = "{}-{}-py3-none-any.whl".format(name, version) %}
+{% set sha256 = "7d39c6ed414993e335fc1c4cbfea62e3f734e9c525bbbab176920ce9dd091d93" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ fn }}
+  url: https://pypi.io/packages/py3/{{ name[0] }}/{{ name }}/{{ fn }}
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py>=34]  
```

Comment:
I think you can make this a `noarch: python` package by dropping the `skip` line here:

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages